code/game/objects/items/devices/radio/radio.dm

```diff
@@ -599,6 +599,17 @@ var/global/list/default_medbay_channels = list(
 	else return
 
 /obj/item/device/radio/emp_act(severity)
+	var/emp
+	on = 0
+	emp++
+	spawn(rand(100, 200))
+		emp--

```

Comment:
I'm guessing you want this to stack; that won't really be the intended effect, since the var is defined on the proc; even if it wasn't temporary, it would just cause multiple spawns and therefore it would tick down in two separate instances, making it, effectively, a single instance, to begin with.